irodsConnector/dataOperations.py

```diff
@@ -321,7 +321,7 @@ def upload_data(self, source: str, destination: irods.collection.iRODSCollection
         if not force:
             space = self.resc_man.resource_space(res_name)
             if size > (space - buff):
-                logging.info(
+                logging.error(
```

Comment:
I think putting the actual word 'ERROR' in the error string is superfluous (several more instances below).

Reply:
All updated.